src/Frontend/Modules/Blog/Layout/Templates/Detail.html.twig

```diff
@@ -0,0 +1,147 @@
+							{% endif %}
+						</li>
+					{% endif %}
+					<li>
+						<a href="{{ item.full_url }}" class="share">{{ lbl.Share|capitalize }}</a>
+					</li>
+				</ul>
+			</header>
+			<div class="bd content" itemprop="articleBody">
+				{% if item.image %}<img src="{{ FRONTEND_FILES_URL }}/blog/images/source/{{ item.image }}" alt="{{ item.title }}" itemprop="image" />{% endif %}
+				{{ item.text|raw }}
+			</div>
+			<footer class="ft">
+				{% if navigation %}
+				<ul class="pageNavigation">
+					{% if navigation.previous %}
+						<li class="previousLink">
+							<a href="{{ navigation.previous.url }}" rel="prev">{{ lbl.PreviousArticle|capitalize }}: {{ navigation.previous.title }}</a>
+						</li>
+					{% endif %}
+					{% if navigation.next %}
+						<li class="nextLink">
+							<a href="{{ navigation.next.url }}" rel="next">{{ lbl.NextArticle|capitalize }}: {{ navigation.next.title }}</a>
+						</li>
+					{% endif %}
+				</ul>
+				{% endif %}
+			</footer>
+		</div>
+	</article>
+
+	{% if comments %}
+		{% if item.allow_comments %}
+			<section id="blogComments" class="mod" itemscope itemtype="http://schema.org/Article">
+				<div class="inner">
+					<header class="hd">
+						<h3 id="{{ act.Comments }}">{{ lbl.Comments|capitalize }}</h3>
+					</header>
+					<div class="bd content">
+						{% for comment in comments %}
+							{# Do not alter the id! It is used as an anchor #}
+							<div id="comment-{{ comment.id }}" class="comment" itemprop="comment" itemscope itemtype="http://schema.org/UserComments">
+								<meta itemprop="discusses" content="{{ item.title }}" />
+								<div class="imageHolder">
+									{% if comment.website %}<a href="{{ comment.website }}">{% endif %}
+										<img src="{{ FRONTEND_CORE_URL }}/Layout/images/default_author_avatar.gif" width="48" height="48" alt="{{ comment.author }}" class="replaceWithGravatar" data-gravatar-id="{{ comment.gravatar_id }}" />
+									{% if comment.website %}</a>{% endif %}
+								</div>
+								<div class="commentContent">
+									<p class="commentAuthor" itemscope itemtype="http://schema.org/Person">
+										{% if comment.website %}<a href="{{ comment.website }}" itemprop="url">{% endif %}
+											<span itemprop="creator name">{{ comment.author }}</span>
+										{% if comment.website %}</a>{% endif %}
+										{{ lbl.Wrote }}
+										<time itemprop="commentTime" datetime="{{ comment.created_on|date('Y-m-d\TH:i:s') }}">{{ comment.created_on|timeago|raw }}</time>
+									</p>
+									<div class="commentText content" itemprop="commentText">
+										{{ comment.text|cleanupplaintext|raw }}
+									</div>
+								</div>
+							</div>
+						{% endfor %}
+					</div>
+				</div>
+			</section>
+		{% endif %}
+	{% endif %}
+	{% if item.allow_comments %}
+		<section id="blogCommentForm" class="mod">
+			<div class="inner">
+				<header class="hd">
+					<h3 id="{{ act.Comment }}">{{ msg.Comment|capitalize }}</h3>
```

Comment:
Wouldn't it be nicer if could do {{ 'act.comment' | trans }} like in Symfony, instead off accessing this array? I guess this should be possible with a filter.